lab_1/main.py

```diff
@@ -11,7 +12,18 @@ def tokenize(text: str) -> list or None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
+    if isinstance(text,str):
```

Comment:
this check can be simplified,
you may add check first and then implement your program without any nesting